Tests/XCUITests/LoginsTests.swift

```diff
@@ -62,6 +62,7 @@ class LoginTest: BaseTestCase {
         mozWaitForElementToExist(passcodeInput, timeout: 20)
         passcodeInput.tap()
         passcodeInput.typeText("foo\n")
+        mozWaitForElementToNotExist(passcodeInput)
```

Comment:
😅 Waiting for the `passcodeInput` to disappear may solve a number of instabilities in the same file.